.scala-steward.conf

```diff
@@ -1,5 +1,7 @@
 updates.ignore = [
   { groupId = "com.github.plokhotnyuk.jsoniter-scala" },
+  { groupId = "ch.epfl.scala", artifactId = "nailgun-server" },
```

Comment:
For some reason I always thought Bloop was using https://github.com/jvican/snailgun not yet another fork of nailgun. I'm curious how much changed in that fork and if the migration off it would be easy.

Reply:
I need to investigate what's the best solution here, especially that the Bloop fork is also using snailgun I think.

Reply:
Actually it does use snailgun :thinking: